File: logisim/app/src/main/java/logisim/file/LoaderException.java
/* Copyright (c) 2010, Carl Burch. License information is located in the
 * logisim.Main source code and at www.cburch.com/logisim/. */

package logisim.file;

public class LoaderException extends RuntimeException {
	private boolean shown;

	LoaderException(String desc) {
		this(desc, false);
	}

	LoaderException(String desc, boolean shown) {
		super(desc);
		this.shown = shown;
	}

	public boolean isShown() {
		return shown;
	}
}

File: LogisimPlus/LogisimPlus/logisim/file/LoaderException.cs
// ====================================================================================================
// Produced by the Free Edition of Java to C# Converter.
// To produce customized conversions, purchase a Premium Edition license:
// https://www.tangiblesoftwaresolutions.com/product-details/java-to-csharp-converter.html
// ====================================================================================================

using System;

/* Copyright (c) 2010, Carl Burch. License information is located in the
 * logisim.Main source code and at www.cburch.com/logisim/. */

namespace logisim.file
{
	public class LoaderException : Exception
	{
		private bool shown;

		internal LoaderException(string desc) : this(desc, false)
		{
		}

		internal LoaderException(string desc, bool shown) : base(desc)
		{
			this.shown = shown;
		}

		public virtual bool Shown
		{
			get
			{
				return shown;
			}
		}
	}
}
